glean-core/src/lib.rs

```diff
@@ -138,15 +138,16 @@ impl Glean {
                     "No content for ping '{}', therefore no ping queued.",
                     ping_name
                 );
-                return Ok(());
+                return Ok(false);
             }
             Some(content) => content,
         };
 
         // FIXME: Logging ping content for now.  Eventually this should be controlled by a flag
         log::info!("{}", ping_content);
```

Comment:
Do we still need this? We now have the `Configuration` on the Kotlin side, which exposes the flag to "logPings".